var/spack/repos/builtin/packages/vvtest/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from os import system
+from spack import *
+
+
+class Vvtest(Package):
+    """Vvtest is a test harness originally authored by Richard Drake for the
+    Alegra-Emphasis project, written in pure Python, and designed for running
```

Comment:
Shouldn't Python be a dependency in this case? What other dependencies are missing?